### PR TITLE
PSRP - Fix docs for wrong var name

### LINKED ISSUE #58714: PSRP documentation has wrong parameter name
##### SUMMARY

The [PSRP documentation](https://docs.ansible.com/ansible/latest/plugins/connection/psrp.html) mentions `ansible_psrp_connection_backoff` but this seems to be incorrect. The code appears to refer to `ansible_psrp_reconnection_backoff` instead (`reconnection` vs `connection`).

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

PSRP
lib/ansible/plugins/connection/psrp.py

##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible-2.8.0.dev0-py2.7.egg/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.15rc1 (default, Nov 12 2018, 14:31:15) [GCC 7.3.0]
```

##### CONFIGURATION

```paste below
None
```

##### OS / ENVIRONMENT

Linux, Ubuntu 18.04.2 LTS

##### ADDITIONAL INFORMATION

None.

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
Fixes: #58714
##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: lib/ansible/plugins/connection/psrp.py [hunk 1]
```diff
@@ -119,7 +119,7 @@
       (First sleeps X, then sleeps 2*X, then sleeps 4*X, ...)
     - This is measured in seconds.
     vars:
-    - name: ansible_psrp_connection_backoff
+    - name: ansible_psrp_reconnection_backoff
```
Comment (jborean93) [inline lib/ansible/plugins/connection/psrp.py:122]: You can't change it now that it's been in a release, you can only just add a new variable and keep the existing one.

Comment (ShachafGoldstein) [reply lib/ansible/plugins/connection/psrp.py:122]: Trying to fix the issue #58714 - seems he's right and the module does refer to a different var

Comment (jborean93) [reply lib/ansible/plugins/connection/psrp.py:122]: The options doc section actually define the vars that are used to control that option. The name of the var can be anything and typically we do just name like `ansible_<connection>_<option>` but that didn't happen in this case.

This means that you can still set `ansible_psrp_connection_backoff` to control this option and this PR makes it so you cannot. We need to ensure that it is backwards compatible which means keeping the older var and just adding a new one called `ansible_psrp_reconnection_backoff`.

Comment (ShachafGoldstein) [reply lib/ansible/plugins/connection/psrp.py:122]: I see. I'm gonna close this PR and see what develops in the originating issue

**Pull request closed without merging**